templates/core/features/secure-boot.html

```diff
@@ -1,150 +1,197 @@
 
 {% block title %}Secure boot | Ubuntu Core{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1ltH7T2boIJ7VcUni4Kmc0XGlmmxWAf5u82LEwkEGP24/edit#{% endblock meta_copydoc %}
 
-{% block content %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1ltH7T2boIJ7VcUni4Kmc0XGlmmxWAf5u82LEwkEGP24/edit#
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Secure boot</h1>
-      <p class="p-heading--4">Protect against vulnerabilities at boot time</p>
-      <p>Computers are vulnerable during the boot process if they are not secured. The kernel, hardware peripherals and user space processes are all initiated at boot and any vulnerability in the boot firmware can have cascading effects on the entire system.</p>
-      <p>In the case of an attack on boot firmware, damages are so profound that often hardware replacement is the only fix. In an industrial IoT scenario, this means considerable downtime, manual maintenance, possibly at several locations and CapEx for hardware replacement. An utterly undesirable outcome.</p>
-      <p>
-        <a href="/core/docs/security-and-sandboxing"><span class="x x-first x-last">Learn about </span>security <span class="x x-first x-last">and sandboxing in </span>Ubuntu <span class="x x-first x-last">Core ›</span></a>
-      </p>
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Secure boot</h1>
+      </div>
+      <div class="col">
+        <p class="p-heading--5">Protect against vulnerabilities at boot time</p>
+        <p>
+          Computers are vulnerable during the boot process if they are not secured. The kernel, hardware peripherals and user space processes are all initiated at boot and any vulnerability in the boot firmware can have cascading effects on the entire system.
+        </p>
+        <p>
+          In the case of an attack on boot firmware, damages are so profound that often hardware replacement is the only fix. In an industrial IoT scenario, this means considerable downtime, manual maintenance, possibly at several locations and CapEx for hardware replacement. An utterly undesirable outcome.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/core/docs/security-and-sandboxing">Learn about security and sandboxing in Ubuntu Core &nbsp;&rsaquo;</a>
+        </p>
+      </div>
     </div>
-    <div class="col-5 u-hide--small u-vertically-center u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/6ed19d21-UC20_Secure_boot.svg",
-        alt="Secure boot",
-        width="190",
-        height="190",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+    <div class="u-fixed-width">
+      <div class="p-image-container--cinematic">
```

Comment:
Could we change this to "p-image-container" and adjust the top padding? I guess this is similar to @mattea-turic's comment but it looks like there is too much white space for the row of images.